app/src/main/java/com/asfoundation/wallet/ui/iab/LocalPaymentFragment.kt

```diff
@@ -467,4 +460,6 @@ class LocalPaymentFragment : DaggerFragment(), LocalPaymentView {
     }
   }
 
+  private fun isAsync() = ASYNC_PAYMENTS_ID.contains(paymentId)
+
```

Comment:
This logic should be on the interact. the presenter would then have the paymentId and ask the interactor if it's async